internal/lg/lg.go

```diff
@@ -0,0 +1,62 @@
+// short for "log"
+package lg
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Logger interface {
+	Output(maxdepth int, s string) error
+}
+
+const (
+	DEBUG = 1
+	INFO  = 2
+	WARN  = 3
+	ERROR = 4
+	FATAL = 5
+)
+
+func ParseLogLevel(levelstr string, verbose bool) (int, error) {
```

Comment:
Yes, I can use a type, but I don't think it helps much.

```go
logger.Output(3, fmt.Sprintf("%s: "+f, msgLevel, args...))
```

doesn't work, go says

```
../../go/src/github.com/nsqio/nsq/internal/lg/lg.go:66: too many arguments in call to fmt.Sprintf
	have (string, LogLevel, []interface {}...)
	want (string, ...interface {})
```

Thus the uglier thing I pushed up.